packages/integration-tests/src/integrationTests/sendMessages.test.ts

```diff
@@ -193,40 +193,40 @@ describe.only('send message - users are online', () => {
   })
 
   test('Owner replicated all messages', async () => {
-    await assertReceivedMessages('owner', 3, 120_000, owner.store)
+    await assertReceivedMessages('owner', 4, 240_000, owner.store)
   })
 
   test('userOne replicated all messages', async () => {
-    await assertReceivedMessages('userOne', 3, 120_000, userOne.store)
+    await assertReceivedMessages('userOne', 4, 240_000, userOne.store)
```

Comment:
Maybe we can keep 240_00 as const somewhere as timeout for receiving messages since they all are the same?

Reply:
good catch, gonna move it to variable